terraform/environments/digital-prison-reporting/application_variables.json

```diff
@@ -3,6 +3,8 @@
     "development": {
       "project_short_id": "dpr",
       "create_database": true,
+      "deploy_operational_database": true,
+      "scale_down_kinesis_agent": false,
```

Comment:
Only deploy the operational DB in the dev environment. Also, turn off scaling down the Kinesis Agent VM in dev so the IP we use to access NOMIS doesn't change whilst data is being loaded.